prExplorer/client-app/public/locales/en/translation.json

```diff
@@ -0,0 +1,5 @@
+{
+    "products": "Products",
```

Comment:
This is a good solution with i18n. However, consider that soon your translations might also be needed to use on the server. For example, for translating some exceptions or maybe you will send emails/notifications from your server and you want them to be translated as well.

In the future you can think of moving these translation files to the server. Then to use these common files both on the client and the server.

In .NET, you can use `IStringLocalizer`. More info here: https://learn.microsoft.com/en-us/dotnet/core/extensions/localization

Reply:
I can create translationcontroller and store these on database, serve as json. And use it on translationStore same as productStore. That would be good

Reply:
Exactly, but also consider using `IStringLocalizer` for the server